File: src/main/webapp/resources/js/controllers/RegisterUserController.js
/**
 * 
 */
var RegisterUserController = function($rootScope, $scope, $http, $location) {

	$scope.RegisterUser = function(user) {
		var req = {
			method : 'POST',
			url : 'RegisterUserController/RegisterUser',
			headers : {
				"Content-Type" : "application/json"
			},
			data : user
		};
		$http(req).success(function(data) {
			if (data == "OK") {
				console.log("Đăng ký thành công!");
				$location.path("/Login");
			} else {
				console.log("Lỗi!");
			}
		}).error(function(error) {
			console.error(error);
		});
	};

};

File: src/main/java/vn/edu/hcmuaf/controller/RegisterUserController.java
package vn.edu.hcmuaf.controller;

import org.springframework.http.HttpHeaders;
import org.springframework.http.HttpStatus;
import org.springframework.http.MediaType;
import org.springframework.http.ResponseEntity;
import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestBody;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;
import org.springframework.web.bind.annotation.ResponseBody;
import org.springframework.web.client.RestTemplate;

import vn.edu.hcmuaf.util.LinkService;

import com.vod.model.User;

@Controller
@RequestMapping("/RegisterUserController")
public class RegisterUserController {

	@RequestMapping(value = "/layout")
	public String doGetLayoutPage() {
		return "register";
	}

	@RequestMapping(value = "/RegisterUser", method = RequestMethod.POST, consumes = "application/json")
	public @ResponseBody
	String doRegisterUser(@RequestBody User user) {
		String result = "FAIL";
		RestTemplate restTemplate = new RestTemplate();
		HttpHeaders headers = new HttpHeaders();
		headers.add("Accept", MediaType.APPLICATION_JSON_VALUE);
		ResponseEntity<Void> entity = restTemplate.postForEntity(
				LinkService.USER_ADD, user, Void.class);
		if (entity.getStatusCode().equals(HttpStatus.OK)) {
			result = "OK";
		}

		return result;
	}
}
